index.html

```diff
@@ -1514,6 +1514,34 @@ <h3>publicKeyPem</h3>
         </pre>
       </section>
 
+      <section id="publicKeyHex" about="https://w3id.org/security#publicKeyHex"
+      typeof="rdf:Property">
+        <h3>publicKeyHex</h3>
+        <p>
+A public key Hex property is used to specify the hex-encoded version of the public key, based on <a href="https://tools.ietf.org/html/rfc4648#section-8">section 8 of rfc4648</a>. Hex encoding is also known as Base16 encoding.
+        </p>
+        <dl>
+          <dt>Status</dt>
+          <dd property="vs:term_status">unstable</dd>
+          <dt>Domain</dt>
+          <dd>Key</dd>
+          <dt>Range</dt>
+          <dd>xsd:string</dd>
+        </dl>
+        <p>
+    The following example demonstrates the expression of a public key
+    in hex format.
+        </p>
+        <pre class="example prettyprint language-jsonld">
+{
+    "id": "did:example:123#hexKey",
+    "controller": "did:example:123",
+    "type": "EcdsaSecp256k1RecoveryMethod2020",
+    "publicKeyHex":"027560af3387d375e3342a6968179ef3c6d04f5d33b2b611cf326d4708badd7770"
```

Comment:
This example is wrong -- it uses lowercase letters, which the IETF RFC you point to does not. :) ... decisions, decisions. :P 

You should just do what the referred to spec says (use uppercase). There will, inevitably, be people that want to do both, so it'll be up to you to fight that battle and not give in. The easy path will be to support upper and lowercase, and if you do that, you'll destroy the deterministic sorting provided by base32hex.

Reply:
Ohhh good catch, let's define it properly here and when I catch people are doing it wrong I'll refer them here to point them in this direction.

Reply:
ahhhh! yes this is a great catch... I gotten beaten up by this issue in my NFC implementation.